filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.filesystems.blob
+
+import com.azure.core.credential.AzureSasCredential
+import cromwell.filesystems.blob.BlobPathBuilder
+import org.scalatest.Ignore
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.nio.file.Files
+
+@Ignore
+class BlobPathBuilderSpec extends AnyFlatSpec with Matchers{
+
```

Comment:
Like we just talked about, let's add a few mosts tests that check the various invalid behaviors.